google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -45,6 +45,8 @@ appengine.appyaml.location.label=app.yaml path\:
 appengine.dockerfile.location.browse.button=Browse for Your Dockerfile Location
 appengine.deployment.error.with.code=Deployment failed with exit code: {0}\nPlease make sure that you are using the latest version of the Google Cloud SDK.\nRun ''gcloud components update'' to update the SDK. (See: https://cloud.google.com/sdk/gcloud/reference/components/update.)
 appengine.deployment.error.during.execution=Deployment failed due to an unexpected error during the deployment execution phase.
+appengine.stopapp.error.during.execution=Failed to stop application due to an unexpected error during the execution phase.
```

Comment:
appengine.stop.module.version.execution.error?

I'm just wondering if we might end up supporting module or app level stop.